packages/twenty-server/src/database/typeorm/metadata/migrations/1708449210922-addOnDeleteActionToRelationMetadata.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddOnDeleteActionToRelationMetadata1708449210922
+  implements MigrationInterface
+{
+  name = 'AddOnDeleteActionToRelationMetadata1708449210922';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TYPE "metadata"."relationMetadata_ondeleteaction_enum" AS ENUM('CASCADE', 'RESTRICT', 'SET NULL')`,
```

Comment:
SET_NULL ? I would prefer not having spaces

Reply:
@charlesBochet I originally had this without spaces but that's how postgresql option looks like so I think it will avoid confusion later (otherwise we have to convert back and forth to remove/add the _)

Reply:
We might have to map SET_NULL to SET NULL while using it though, just felt weird in the db, I'm use to upper snake case for enums

Reply:
Ah, I didn't know that. From my understand, It's already what prisma is doing and I see they are planning to revert back because there is no strong reason to do it since it's allowed by the engine (and people opened a few issues about it). Anyway, I see that it's concerning you both so I'll update my code 😛